pkg/client/unversioned/clientcmd/auth_loaders.go

```diff
@@ -74,7 +74,7 @@ func (a *PromptingAuthLoader) Prompt() *clientauth.Info {
 }
 
 func promptForString(field string, r io.Reader) string {
-	fmt.Printf("Please enter %s: ", field)
+	fmt.Printf("Please enter %s: \n", field)
```

Comment:
I don't think we need `\n` here, since having user input right after `: ` looks nice:

```console
$ Please enter Username: <user_input>
$ Please enter Password: <user_input>
```